views/chartTest.pug

```diff
@@ -24,14 +24,12 @@ block content
 
       var socket = io.connect('//'+document.location.hostname+':'+document.location.port);
       socket.on('chartData', function (data) {
-        console.log(data);
-
         window.chart.push([{
           time: data.time,
           y: data.value
         }]);
-        socket.emit('dataReceived', { data: 'received the data' });
       });
+      socket.emit('subscribeToID', { deviceId: #{deviceId} });
```

Comment:
This should be in quotations I think the expectation is that the device data is a string

socket.emit('subscribeToID', { deviceId: '#{deviceId}' });